### PR TITLE
add monitorResourceType variable to monitor return resource

### LINKED ISSUE #497: Monitor - Add MonitorResourceType variable to monitor return resource
We should keep the resource names of monitor input parameters and returned results consistent. For example, if the input parameter is "cpu", the return parameter should also be "cpu". HuaweiCloud and FlexibleEngine currently needs to be modified.

### PR DESCRIPTION
**Pull request by jinyangyang222**
fixed #497

### CODE REVIEW

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/monitor/Metric.java [hunk 1]
```diff
@@ -30,6 +31,10 @@ public class Metric {
     @Schema(description = "The type of the metric.")
     MetricType type;
 
+    @NotNull
+    @Schema(description = "The resources type of the monitor.")
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/monitor/Metric.java:35]: @jinyangyang222 description should be "the resource type  of the metric."

Comment (jinyangyang222) [reply modules/models/src/main/java/org/eclipse/xpanse/modules/models/monitor/Metric.java]: ok, fixed.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/monitor/Metric.java [hunk 2]
```diff
@@ -30,6 +31,10 @@ public class Metric {
     @Schema(description = "The type of the metric.")
     MetricType type;
 
+    @NotNull
+    @Schema(description = "The resource type of the metric.")
+    MonitorResourceType monitorResourceType;
+
     @NotNull
     @Schema(description = "The unit of the metric.")
     MetricUnit unit;
```
[the remark this answers is not in the document]
Comment (swaroopar) [reply modules/models/src/main/java/org/eclipse/xpanse/modules/models/monitor/Metric.java:40]: @sonatype-lift ignore

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**